packages/sfui/frameworks/react/components/SfSelect/SfSelect.tsx

```diff
@@ -33,6 +63,10 @@ export default function SfSelect(props: SfSelectProps) {
           className,
         )}
         data-testid="select-input"
+        onBlur={blurSelect}
+        onChange={changedValue}
+        onClick={clickSelect}
+        onKeyDown={keyDownSelect}
```

Comment:
What do you think about changing the names of handlers to `blurHandler`, `clickHandler`, `keydownHandler`?

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2574/commits/7b24a1ac2af1ba87cb4eb7fa423429edb324cce0